pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools>=64.0.1", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "libssa"
+dynamic = ["version"]
+authors = [
+    { name = "Kleydson Stenio" },
+]
+description = "Put a short project description here"
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+requires-python = ">3.8,<3.11"
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
I added 3.10 because I use 3.10 locally and it seems to work. I did not test everything, though.